scripts/build_dists_archive.sh

```diff
@@ -199,53 +206,70 @@ echo "Hash file $hashFile"
+cp -f "${PWD}/target/release/tari_mining_node" "${distDir}/runtime/tari_mining_node"
+
+# Merge Mining Proxy
+cp -f -P "${PWD}/applications/tari_merge_mining_proxy/${sDist}/start_tari_merge_mining_proxy" "${distDir}/start_tari_merge_mining_proxy"
+cp -f -P "${PWD}/applications/tari_merge_mining_proxy/${sDist}/start_xmrig" "${distDir}/start_xmrig"
+cp -f "${PWD}/applications/tari_merge_mining_proxy/${sDist}/runtime/start_tari_merge_mining_proxy.sh" "${distDir}/runtime/start_tari_merge_mining_proxy.sh"
+cp -f "${PWD}/applications/tari_merge_mining_proxy/${sDist}/runtime/start_xmrig.sh" "${distDir}/runtime/start_xmrig.sh"
+cp -f "${PWD}/target/release/tari_merge_mining_proxy" "${distDir}/runtime/tari_merge_mining_proxy"
+
+# 3rd party install
+cp -f "${PWD}/buildtools/install_xmrig.sh" "${distDir}/runtime/install_xmrig.sh"
+
+# Config
+cp -f "${PWD}/common/config/presets/tari_sample.toml" "${distDir}/config/config.toml"
+cp -f "${PWD}/common/xmrig_config/config_example_stagenet.json" "${distDir}/config/xmrig_config_example_stagenet.json"
+cp -f "${PWD}/common/xmrig_config/config_example_mainnet.json" "${distDir}/config/xmrig_config_example_mainnet.json"
+cp -f "${PWD}/common/xmrig_config/config_example_mainnet_self_select.json" "${distDir}/config/xmrig_config_example_mainnet_self_select.json"
+
+pushd ${distDir}
 if [ "$osname" == "osx" ]  && [ -n "${osxsign}" ]; then
   echo "Setup OSX Binaries signing ..."
-  for SIGN_FILE in $(find "${distDir}/dist" -maxdepth 1 -name "tari_*" -type f -perm +111 ); do
+  for SIGN_FILE in $(find "${distDir}/runtime" -maxdepth 1 -name "tari_*" -type f -perm +111 ); do
     echo "Signing OSX Binary - $SIGN_FILE ..."
     codesign --options runtime --force --verify --verbose --sign "${osxsign}" "$SIGN_FILE"
     echo "Verify signed OSX Binary - $SIGN_FILE ..."
     codesign --verify --deep --display --verbose=4 "$SIGN_FILE"
     spctl -a -v "$SIGN_FILE"
   done
 fi
-shasum -a $shaSumVal * >> "$distDir/$hashFile"
-#echo "$(cat $distDir/$hashFile)" | shasum -a $shaSumVal --check --status
-echo "$(cat $distDir/$hashFile)" | shasum -a $shaSumVal --check
-mv "$distDir/$hashFile" "$distDir/dist/"
-#tar -cjpf "$distDir/$archiveFile" .
-zip -j "$distDir/$archiveFile" *
-cd ..
-shasum -a $shaSumVal "$archiveFile" >> "$distDir/$archiveFile.sha${shaSumVal}sum"
-#echo "$(cat $distDir/$archiveFile.sha${shaSumVal}sum) $distDir/$archiveFile" | shasum -a $shaSumVal --check
-echo "$(cat $distDir/$archiveFile.sha${shaSumVal}sum)" | shasum -a $shaSumVal --check
+shasum -a ${shaSumVal} * >> "${distDir}/${hashFile}"
+
+echo "$(cat ${distDir}/${hashFile})" | shasum -a ${shaSumVal} --check
+
+zip -r "${distDir}/${archiveFile}" *
+echo ${distDir}/${archiveFile}
+shasum -a ${shaSumVal} "${archiveFile}" >> "${distDir}/${archiveFile}.sha${shaSumVal}sum"
+
+echo "$(cat ${distDir}/${archiveFile}.sha${shaSumVal}sum)" | shasum -a ${shaSumVal} --check
 popd
-echo "Delete $distDir"
```

Comment:
I would leave this in, as a reminder to clean out the temp folder that is created. Thou most Nix OSs do a temp clean up on every reboot. If you not worried about space or debugging, then just remove the temp folder with the ```rm -fr $distDir```

Reply:
Re-added